languages/rust/exercises/concept/string-vs-str/.meta/example.rs

```diff
@@ -0,0 +1,35 @@
+pub struct CSVBuilder {
```

Comment:
```suggestion
pub struct RecordBuilder {
```

Nit: this should build a single record, not a full CSV file. Also, keeps it consistent with the instructions.